public/docs/ts/latest/guide/displaying-data.jade

```diff
@@ -17,42 +17,42 @@ figure.image-display
   img(src="/resources/images/devguide/displaying-data/final.png" alt="Final UI")
 
 :marked
-  # Table Of Contents
+  # Contents
```

Comment:
Do we usually use a first-level heading for Contents? (I'm actually not sure. I tend not to in my own writing, but I don't remember whether we've established a good precedent in these pages.)